README.md

```diff
@@ -92,6 +92,7 @@ A curated list of awesome [Apache Spark](https://spark.apache.org/) packages and
 * [KeystoneML](http://keystone-ml.org/) - Type safe machine learning pipelines with RDDs.
 * [JPMML-Spark](https://github.com/jpmml/jpmml-spark) - PMML transformer library for Spark ML.
 * [Distributed Keras](https://github.com/cerndb/dist-keras) - Distributed deep learning framework with PySpark and Keras.
+* [ModelDB](https://mitdbg.github.io/modeldb) - A system to manage machine learning models
```

Comment:
Found this video https://www.youtube.com/watch?v=gxBb4CjJcxQ&feature=youtu.be
Maybe you can submit a Python 3 upgrade